### DISCUSSION TITLE
More than one inverter?

### DISCUSSION THREAD

**Discussion by drsmarsden** [hultenvp/solis-sensor#150] [Q&A]
Hi

I am thinking of adding a vertical wind Turbine , which would mean a 2nd inverter

would they both integrate with HA ?

Stuart

**Comment by hultenvp** [comment]
Hi Stuart,

Yes, that's no issue at all. I have 2 inverters of different brands myself and it's working fine:
<img width="961" alt="image" src="https://user-images.githubusercontent.com/61835400/187034720-2efbb819-f1e3-4a11-a181-0dc1a1fbecd9.png">

Same brand is also no problem by the way. Both cases you just add a second config to the yaml.

Cheers
